### ISSUE TITLE
Cannot read property xs of undefined

### ISSUE THREAD

**Issue by teofilp** [ml5js/ml5-examples#275]
Hello guys, so what i'm trying to do is train a model to predict some energy consumption given a date. But when i try to normalize the model i get the following error: 
"Cannot read property 'xs' of undefined
    at t.value (NeuralNetworkData.js:179)
    at t.value (NeuralNetworkData.js:370)"
data = await fetch("http://localhost:8080/energy.json").then((res) => {
        return res.json();
    });

    // console.log(data);
    let options = {
        inputs: ['day', 'month', 'year'],
        output: ["0:00", "0:30", "1:00", "1:30", "2:00", "2:30", "3:00", "3:30", "4:00", "4:30", "5:00", "5:30", "6:00", "6:30", "7:00", "7:30", "8:00", "8:30", "9:00", "9:30", "10:00", "10:30", "11:00",
            "11:30", "12:00", "12:30", "13:00", "13:30", "14:00", "14:30", "15:00", "15:30",
            "16:00", "16:30", "17:00", "17:30", "18:00", "18:30", "19:00", "19:30", "20:00", "20:30", "21:00", "21:30", "22:00", "22:30", "23:00", "23:30", "Total"
        ],
        task: 'classification',
        debug: true
    }

    model = ml5.neuralNetwork(options);

    for (let i = 0; i < data.length; i++) {
        let rec = data[i];

        let inputs = {
            xs: rec.xs
        }

        let targets = {
            ys: rec.ys
        }
        model.addData(
            inputs, targets
        );
    }

    let trainOptions = {
        epochs: 200,
        batchSize: 400
    }

    model.data.normalize();

    model.train(trainOptions, (epoch, loss) => {
        console.log(loss);
    }, () => {
        console.log('training finished');
    });

**Comment by teofilp** [comment]
my mistake, i should've not set targets.ys but instead dirrectly assign targets
